redisinsight/ui/src/utils/formatters/valueFormatters.tsx

```diff
@@ -108,16 +110,30 @@ const formattingBuffer = (
       }
     }
     case KeyValueFormat.Vector32Bit: {
+      const utfVariant = bufferToUTF8(reply)
       try {
+        if (props?.isField) return { value: utfVariant, isValid: true }
+        const bufferFromUtf = Buffer.from(utfVariant)
+        const bufferFromValue = Buffer.from(reply.data)
+        if (buffersEqual(bufferFromUtf, bufferFromValue)) {
```

Comment:
we already have 'isEquelBuffers' util

Reply:
removed buffersEqual function, made a little change in isEqualBuffer